### ISSUE TITLE
Add Unified End Repair Python Script

### ISSUE THREAD

**Issue by LeeBergstrand** [jmtsuji/rotary#58]
Unify, Bug Fix, and improve end repair by having it implemented as a python script. This work is currently happening on: https://github.com/jmtsuji/rotary/tree/end_repair

**Comment by jmtsuji** [comment]
The updated `repair.py` (called from the command line as `rotary-repair`) is now working in the rotary pipeline! I ran an end-to-end test that seems to have run fine through the assembly section of the workflow (although see notes at the bottom).

I had to add four assembly-related tools to the [`environment.yaml`](https://github.com/jmtsuji/rotary/blob/end_repair/enviroment.yaml) file to be able to run `rotary-repair`. One of these, `circlator`, is no longer being actively maintained and has a huge number of dependencies that aren't needed by this project. My long-term hope is to replace `circlator` with in-house code as discussed in #26 -- this should simplify the rotary install substantially. However, the other three tools will need to remain in the environment.yaml.

I think that this end_repair branch is tentatively OK to merge into develop, so I will make a PR for you to review.

Note: When I ran the end-to-end test, I tested commit [9a4678b](https://github.com/jmtsuji/rotary/commit/9a4678b9a03865c2f295079feedc368139fce0a2), which is one commit before what is currently the latest commit, [fe7481f](https://github.com/jmtsuji/rotary/commit/fe7481f2ec0470eb7eb44f43944cc8bf028ee2ab). Commit 9a4678b did not yet have the rotary-repair dependencies as part of environment.yaml, and I instead manually installed rotary-repair in the conda env that was made for the end repair step. In fe7481f, I got rid of the conda env for the end repair step and moved the dependencies to environment.yaml. Because I had to head home at that point and didn't have access to a linux server, I haven't been able to test fe7481f yet to confirm everything is working, although the couple tests I have run locally on my Mac seem to work fine.
